src/queries/update/UpdateReconstructAppContext.ts

```diff
@@ -1,13 +1,14 @@
 import { AppSettings } from "../../config/Variables";
 import { qb } from "../QueryBuilder";
 import { initElements, parsePrefix } from "../../function/FunctionEditVars";
-import { getWorkspaceContextIRI } from "../../function/FunctionGetVars";
 import { processQuery } from "../../interface/TransactionInterface";
 import { updateWorkspaceContext } from "./UpdateMiscQueries";
 import { updateProjectElement } from "./UpdateElementQueries";
 
 export async function reconstructApplicationContextWithDiagrams(): Promise<string> {
   const linkPredicates = [
+    parsePrefix("a-popis-dat-pojem", "odkazuje-na-assetový-kontext"),
+    parsePrefix("a-popis-dat-pojem", "odkazuje-na-přílohový-kontext"),
```

Comment:
I think that only one of the two is valid.

Reply:
The "assetový" predicates are relics from when the first milestone was being implemented. They're removed now.